py/server/tests/test_dbc.py

```diff
@@ -32,6 +34,15 @@ def test_read_adbc(self):
                 # instead of the actual size
                 self.assertEqual(table.size, 10)
 
+    @unittest.skipIf(platform.machine() != "x86_64", reason="connectorx not available on Linux/Arm64"
+                                                            "https://github.com/sfu-db/connector-x/issues/386")
```

Comment:
How does this work? Is this ticket part of 'reason'? Not sure I've seen this pattern used before.

Reply:
yes, that's just concatenation of two literal strings.